pages/committee-members.html.haml

```diff
@@ -71,16 +70,15 @@
       %a{href: 'https://github.com/amanda08'} GitHub
 
   %article
-    %img{src: '/images/committee-members/leonard-garvey.jpg'}
+    %img{src: '/images/committee-members/lauren-hennnessy.jpg'}
```

Comment:
I spent way too long trying to work out why the image didn't load and then I finally worked out the file name has a typo :D